Sources/Public/Pretty.swift

```diff
@@ -305,7 +305,14 @@ extension Pretty {
         option: Option
     ) {
         let plainString = printer(label, targets, separator, option)
-        let coloredString = printer(label, targets, separator, option)
+
+        let coloredOption: Option = {
+            var op = option
+            op.colored = true
+            return op
+        }()
```

Comment:
Wouldn't a code like the following suffice?

```suggestion
        let coloredOption = option
        coloredOption.colored = true
```

Reply:
Swift compiler says for the suggested above: 
> cannot assign to property: 'coloredOption' is a 'let' constant

`var` is ok:

```suggest
var coloredOption = option
coloredOption.colored = true
```

Reply:
Oops, I wrong.
Use `var` instead of `let` is corrected.

Reply:
@YusukeHosonuma 
I've refactored in 7582ec541c146644aad29090a88044299f0b839b.